tests/regression/roles/sonic_dcbx/defaults/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+ansible_connection: httpapi
+module_name: dcbx
+
+tests:
+  - name: test_case_merged_01
+    description: Disable DCBx Interfaces Configurations
+    state: merged
+    input:
+      interfaces:
+        - name: '{{ interface1 }}'
+          enabled: False
+          pfc_tlv_enabled: False
+          ets_configuration_tlv_enabled: False
+          ets_recommendation_tlv_enabled: False
+
+  - name: test_case_merged_02
+    description: Enable global DCBx Configuration
+    state: merged
+    input:
+      global:
+        enabled: True
+
+  - name: test_case_merged_03
+    description: Enable DCBx Configurations on Interfaces 
+    state: merged
+    input:
+      interfaces:
+        - name: '{{ interface1 }}'
+          enabled: True
+          pfc_tlv_enabled: True
+          ets_configuration_tlv_enabled: True
+          ets_recommendation_tlv_enabled: True
+
+  - name: test_case_merged_04
+    description: Disable DCBx Configurations on Interfaces
+    state: merged
+    input:
+      interfaces:
+        - name: '{{ interface1 }}'
+          enabled: False
+          pfc_tlv_enabled: False
+          ets_configuration_tlv_enabled: False
+          ets_recommendation_tlv_enabled: False
+
+  - name: test_case_replaced_01
+    description: Disable global DCBx configuration
+    state: replaced
+    input:
+      global:
+        enabled: False
+  
+  - name: test_case_replaced_02
+    description: Enable DCBx Configurations on Interfaces
+    state: merged
+    input:
+      interfaces:
+        - name: '{{ interface1 }}'
```

Comment:
This would be a more useful test case if it only enabled one or two of the options to ensure that this causes the other options to get "deleted", resulting in enabling them all.

Reply:
Updated